recipes/vulkan-utility-libraries/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,7 @@
+#include <vulkan/layer/vk_layer_settings.hpp>
+#include <iostream>
+
+int main() {
```

Comment:
```suggestion
int main() {
    VkuLayerSettingSet layerSettingSet = VK_NULL_HANDLE;
    vkuCreateLayerSettingSet("VK_LAYER_LUNARG_conan", nullptr, nullptr, nullptr, &layerSettingSet);
```
Let's consume a method at least to check the library.